plugins/cairo-lang-macro-stable/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "cairo-lang-macro-stable"
+version = "1.0.0"
+edition.workspace = true
+
+authors.workspace = true
+categories = ["development-tools"]
+description = "Cairo procedural macro stable ABI interface primitives."
+homepage.workspace = true
+keywords = ["scarb"]
+license.workspace = true
+readme = "README.md"
+repository.workspace = true
+
+[dependencies]
+anyhow.workspace = true
+serde.workspace = true
+serde_json.workspace = true
```

Comment:
None of these seem to be used. Moreover, any dependencies may influence degree of stability we might provide